polymer.externs.js

```diff
@@ -1094,3 +1094,26 @@ Polymer.Iconset = function() {};
  */
 Polymer.Iconset.prototype.applyIcon = function(
       element, icon, theme, scale) {};
+
+Polymer.ResolveUrl = {};
+
+/**
+ * @param {string} cssText Some CSS text taken from ownerDocument.
+ * @param {Document} ownerDocument The source of the css.
```

Comment:
I know we aren't consistent about this in this file, but here and elsewhere, please be explicit about nullability. If it's not nullable `!Document`, if it is, `?Document`.